base/base.php

```diff
@@ -105,7 +105,7 @@ function siteorigin_widget_get_font($font_value) {
 			$font['weight'] = $font_parts[1];
 			$font_url_param .= ':' . $font_parts[1];
 		}
-		$font['css_import'] = '@import url(http' . ( is_ssl() ? 's' : '' ) . '://fonts.googleapis.com/css?family=' . $font_url_param . ');';
+		$font['css_import'] = '@import url(http' . ( strpos( site_url(), 'https' ) !== false ? 's' : '' ) . '://fonts.googleapis.com/css?family=' . $font_url_param . ');';
```

Comment:
We should just always use `https` as Google will "always" serve their assets over `https`. It removes the complexity and any potential bugs. And it won't cause any issues to use an `https` request on a site without SSL enabled.